asst1/asst1.py

```diff
@@ -0,0 +1,104 @@
+from collections import Counter
```

Comment:
Aight just gotta say absolutely LOVE your github name but v unclear as to who you are